clap_complete/src/shells/bash.rs

```diff
@@ -64,15 +64,15 @@ complete -F _{name} -o nosort -o bashdefault -o default {name}
                 cmd = bin_name.replace('-', "__"),
```

Comment:
imo we should single-source the top-level value much like we do in `add_command` for the nested subcommand values

This would involve
- rename this to `fn_name`
- create this as a variable before the `format!`
- pass `fn_name` as a new `parent_fn_name` parameter to `all_subcommands`, replacing the `fn_name` constructed inside of it